### ISSUE TITLE
AppVeyor: NumPy detected, but numpy/arrayobject.h not found

### ISSUE THREAD

**Issue by mloskot** [PDAL/PDAL#1165]
Currently, on AV, one job is constantly failing:

https://ci.appveyor.com/project/hobu/pdal/build/1.0.1330/job/1ese8lj0c0dr8gkq#L728

Apparently, due to NumPy issue: Python and NumPy are reported as found:
```
-- The following REQUIRED packages have been found:
 
 * ZLIB
   Compression support in BPF
 * GEOS (required version >= 3.3)
   Provides general purpose geometry support
   Enables attribute filter
 * GDAL (required version >= 1.9.0)
   Provides general purpose raster, vector, and reference system support
   Enables attribute filter
 * Threads , The thread library of the system
 * PythonInterp
 * PythonLibs
 * NumPy (required version >= 1.5)
```

but `numpy/arrayobject.h` is not found:

```
C:\projects\pdal\src\plang\Array.cpp(53): fatal error C1083: Cannot open include file: 'numpy/arrayobject.h': No such file or directory [C:\projects\pdal\src\plang\pdal_plang.vcxproj]
C:\projects\pdal\src\plang\Invocation.cpp(51): fatal error C1083: Cannot open include file: 'numpy/arrayobject.h': No such file or directory [C:\projects\pdal\src\plang\pdal_plang.vcxproj]
C:\projects\pdal\src\plang\Environment.cpp(43): fatal error C1083: Cannot open include file: 'numpy/arrayobject.h': No such file or directory [C:\projects\pdal\src\plang\pdal_plang.vcxproj]
```

-----

The problem found thanks to Feador@AV: http://help.appveyor.com/discussions/problems/4227-build-prematurely-terminates-command-exited-with-code-1
Since we run the AV builds in parallel mode, it is important that in case of any mysterious failures are inspected scrolling up the log - errors are not guaranteed to be displayed at the bottom.

**Comment by abellgithub** [comment]
Any idea what might have changed to cause this issue?

On Fri, Feb 26, 2016 at 8:11 PM, Mateusz Łoskot <notifications@github.com>
wrote:

> Currently, on AV, one job is constantly failing:
>
>
> https://ci.appveyor.com/project/hobu/pdal/build/1.0.1330/job/1ese8lj0c0dr8gkq#L728
>
> Apparently, due to NumPy issue: Python and NumPy are reported as found:
>
> -- The following REQUIRED packages have been found:
>
>  * ZLIB
>    Compression support in BPF
>  * GEOS (required version >= 3.3)
>    Provides general purpose geometry support
>    Enables attribute filter
>  * GDAL (required version >= 1.9.0)
>    Provides general purpose raster, vector, and reference system support
>    Enables attribute filter
>  * Threads , The thread library of the system
>  * PythonInterp
>  * PythonLibs
>  * NumPy (required version >= 1.5)
>
> but numpy/arrayobject.h is not found:
>
> C:\projects\pdal\src\plang\Array.cpp(53): fatal error C1083: Cannot open include file: 'numpy/arrayobject.h': No such file or directory [C:\projects\pdal\src\plang\pdal_plang.vcxproj]
> C:\projects\pdal\src\plang\Invocation.cpp(51): fatal error C1083: Cannot open include file: 'numpy/arrayobject.h': No such file or directory [C:\projects\pdal\src\plang\pdal_plang.vcxproj]
> C:\projects\pdal\src\plang\Environment.cpp(43): fatal error C1083: Cannot open include file: 'numpy/arrayobject.h': No such file or directory [C:\projects\pdal\src\plang\pdal_plang.vcxproj]
>
> ------------------------------
>
> The problem found thanks to Feador@AV:
> http://help.appveyor.com/discussions/problems/4227-build-prematurely-terminates-command-exited-with-code-1
> Since we run the AV builds in parallel mode, it is important that in case
> of any mysterious failures are inspected scrolling up the log - errors are
> not guaranteed to be displayed at the bottom.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/PDAL/PDAL/issues/1165>.
>



-- 
Andrew Bell
andrew.bell.ia@gmail.com

**Comment by mloskot** [comment]
Bloody hell :) I had no idea environment set in appveyor.xml is used in the BAT scripts. 
I renamed OSGEODIR to OSGEO4W_ROOT and missed to update config.cmd.
OSGEO4W_ROOT is de-facto 'standard' location used in Find*.cmake modules, so I aimed to enable it.

I hope it's been fixed now with 009029149e8ecb6bb6032746d1564abad894e1b4

**Comment by mloskot** [comment]
Fixed https://ci.appveyor.com/project/hobu/pdal/build/1.0.1355
